kubenow/gluster-s3/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for a Gluster-S3 deployment.
+# this is a yaml-formatted file.
+# declare variables to be passed into your templates.
+releaseName: s3-interface
+secretName: s3-secret
+
+account: account
```

Comment:
what is this?

Reply:
Default s3 account. users may want to deploy the chart with default settings for testing or so.